package.json

```diff
@@ -15,9 +24,15 @@
     "eslint-plugin-import": "^2.8.0",
     "eslint-plugin-jsx-a11y": "^6.0.3",
     "eslint-plugin-prettier": "^2.5.0",
-    "eslint-plugin-react": "^7.5.1"
+    "eslint-plugin-react": "^7.5.1",
+    "prettier": "^1.10.2"
   },
   "peerDependencies": {
     "eslint": "^4.16.0"
+  },
+  "devDependencies": {
+    "eslint": "^4.16.0",
+    "husky": "^0.14.3",
```

Comment:
should not all devDependencies here go in dependencies? How are we differentiating the two?

Reply:
No, the `devDependencies` are, as the name suggests, only relevant for working on the project. `husky` and `lint-staged` are only needed for formatting commits, and we don't want to add `eslint` to dependencies because then that would prevent consuming apps from choosing their own version.